### DISCUSSION TITLE
米家游戏第一次武器池没歪

### DISCUSSION THREAD

**Discussion by happyfucking** [share121/inter-knot#4509] [灌水]
![Screenshot_20240713_141900](https://github.com/user-attachments/assets/e29826cd-b1c4-4cd5-adda-9b2cb4d6ee8e)
![Screenshot_20240713_141855](https://github.com/user-attachments/assets/907b9eff-02b8-4397-8863-6d88708a50bf)
ヾ(^▽^*)))

**Comment by happyfucking** [comment]
(@[]@!!)才发现名字有些不文明，但也第一次不会改见谅，日后找机会学一下改

**Comment by hk0852** [comment]
其实要在github上改，实际上就是改你的github的用户名
